app/scripts/views/form.js

```diff
@@ -166,7 +166,14 @@ define(function (require, exports, module) {
       // applied, no submit handler is fired, and the form validation does not
       // take place.
       if (this.isFormEnabled()) {
-        this.$('button[type=submit]').addClass('disabled');
+        if (this.isInExperimentGroup && this.isInExperimentGroup('disabledButtonState', 'treatment')) {
+          this.logEvent('exp.dbs.treatment.render');
```

Comment:
I added some console logging to metrics.logEvent; this event is logged on every key-press.

> 1501163345958 event: exp.dbs.treatment.render raven.js:51:13
1501163348173 event: exp.dbs.treatment.render raven.js:51:13
1501163348883 event: exp.dbs.treatment.render raven.js:51:13
1501163381889 event: exp.dbs.treatment.render raven.js:51:13
1501163381967 event: exp.dbs.treatment.render raven.js:51:13
1501163382070 event: exp.dbs.treatment.render raven.js:51:13
1501163382229 event: exp.dbs.treatment.render raven.js:51:13